self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryInvoices.java

```diff
@@ -266,12 +273,6 @@ public boolean isPaid(final int id) {
                 throw new IllegalStateException("Invoice with id"
                     + id + " is not part of the contract.");
             }
-            return storage.invoices().tasks(id);
-        }
-
-        @Override
-        public Iterator<Invoice> iterator() {
-            return contractInvoices.get().iterator();
         }
```

Comment:
@criske Why remove the iterator?

Reply:
Is not removed, it will fail the build if so; was just displaced I guess.
https://github.com/self-xdsd/self-core/blob/b23b0cf6e44fa481ce403a926a44de8bb251cdac/self-core-impl/src/test/java/com/selfxdsd/core/mock/InMemoryInvoices.java